src/Controls/tests/TestCases.HostApp/Issues/Issue18452.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System.Net;
+
+namespace Maui.Controls.Sample.Issues
+{
+	[Issue(IssueTracker.Github, 18452, "NullReferenceException throws on Windows when setting Cookies on .NET MAUI WebView", PlatformAffected.UWP)]
+	public class Issue18452 : TestContentPage
+	{
+		protected override void Init()
+		{
+			Grid grid = new Grid();
+			grid.RowDefinitions.Add(new RowDefinition { Height = new GridLength(0.8, GridUnitType.Star) });
+			grid.RowDefinitions.Add(new RowDefinition { Height = new GridLength(0.2, GridUnitType.Star) });
+			WebView webView = new WebView();
+
+			Label label = new Label();
+			label.AutomationId = "Label";
+
+			const string url = "https://httpbin.org/#/Cookies/get_cookies";
```

Comment:
Can we use a msft or dotnet url here?

Reply:
> Can we use a msft or dotnet url here?
Hi @PureWeen ,
We have used the Microsoft Learn documentation URL: https://learn.microsoft.com/en-us/dotnet/.